### PR TITLE
9.0.0 - Changelog and Readme update

### PR DESCRIPTION
**Pull request by Kritika-Bhateja-03**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### OUTPUT

```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

### CODE REVIEW

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

File: docs/ATTRIBUTION.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+OpenSource Licenses for Ansible Provider for OpenManage Enterprise
```
Comment (rajshekarp87) [inline docs/ATTRIBUTION.md]:
```suggestion
OpenSource Licenses for Dell OpenManage Ansible Modules
```

Comment (rajshekarp87) [reply docs/ATTRIBUTION.md]: Also add an entry as "Attribution" under "Table of Contents" of README.md

**Review by rajshekarp87 — CHANGES_REQUESTED** [review]

File: README.md
Comment (sachin-apa) [inline README.md:1]: Lets add badge to show the code coverage of collections branch
[codecov](https://app.codecov.io/gh/dell/dellemc-openmanage-ansible-modules)

Comment (Kritika-Bhateja-03) [reply README.md:1]: added

File: CHANGELOG.rst [hunk 2]
```diff
@@ -4,6 +4,43 @@ Dell OpenManage Ansible Modules Release Notes
 
 .. contents:: Topics
 
+v9.0.0
+======
+
+Release Summary
+---------------
+- Module ``idrac_diagnostics`` is added to run and export diagnostics on iDRAC.
+- Role ``idrac_user`` is added to manage local users for iDRAC.
```
Comment (rajshekarp87) [inline CHANGELOG.rst]:
```suggestion
- Role ``idrac_user`` is added to manage local users of iDRAC.
```

File: CHANGELOG.rst [hunk 3]
```diff
@@ -4,6 +4,43 @@ Dell OpenManage Ansible Modules Release Notes
...
+
+Major Changes
+-------------
+
+- idrac_user - This role is introduced to manage local users for iDRAC.
```
Comment (rajshekarp87) [inline CHANGELOG.rst]:
```suggestion
- idrac_user - This role is introduced to manage local users of iDRAC.
```

File: CHANGELOG.rst [hunk 4]
```diff
@@ -4,6 +4,43 @@ Dell OpenManage Ansible Modules Release Notes
...
+--------
+
+- ome_device_network_services - Issue(212681) - The module does not provide a proper error message if unsupported values are provided for the following parameters- port_number, community_name, max_sessions, max_auth_retries, and idle_timeout.
+- ome_device_power_settings - Issue(212679) - The module displays the following message if the value provided for the parameter ``power_cap`` is not within the supported range of 0 to 32767, ``Unable to complete the request because PowerCap does not exist or is not applicable for the resource URI.``
+- idrac_network_attributes - Issue(279049) -  If unsupported values are provided for the parameter ``ome_network_attributes``, then this module does not provide a correct error message.
+
+Known Issues
+------------
+
+- idrac_diagnostics - Issue(285322) - This module doesn't support ExportePSADiagnosticsResult with SOCKS Proxy.
```
Comment (rajshekarp87) [inline CHANGELOG.rst]:
```suggestion
- idrac_diagnostics - Issue(285322) - This module doesn't support export of diagnostics file to HTTP and HTTPS share via SOCKS proxy.
```

File: CHANGELOG.rst [hunk 5]
```diff
@@ -4,6 +4,43 @@ Dell OpenManage Ansible Modules Release Notes
...
+- idrac_firmware - Issue(279282) - This module does not support firmware update using HTTP, HTTPS, and FTP shares with authentication on iDRAC8.
+- ome_diagnostics - Issue(279193) - Export of SupportAssist collection logs to the share location fails on OME version 4.0.0.
+- ome_smart_fabric_uplink - Issue(186024) - The module supported by OpenManage Enterprise Modular, however it does not allow the creation of multiple uplinks of the same name. If an uplink is created using the same name as an existing uplink, then the existing uplink is modified.
+
+New Modules
+-----------
+
+- dellemc.openmanage.idrac_diagnostics - This module allows to to run and export diagnostics on iDRAC.
```
Comment (rajshekarp87) [inline CHANGELOG.rst:39]: Add a line space

File: CHANGELOG.rst [hunk 6]
```diff
@@ -4,6 +4,43 @@ Dell OpenManage Ansible Modules Release Notes
 
 .. contents:: Topics
 
+v9.0.0
```
Comment (rajshekarp87) [inline CHANGELOG.rst:7]: Update changelogs/changelog.yaml file

**Review by rajshekarp87 — CHANGES_REQUESTED** [review]

**Review by sachin-apa — APPROVED** [review]

**Review by rajshekarp87 — APPROVED** [review]

**Review by sachin-apa — APPROVED** [review]

**Pull request merged**